src/pages/PullRequestPage/PullRequestPageTabs/PullRequestPageTabs.spec.jsx

```diff
@@ -82,22 +107,40 @@ afterAll(() => {
 
 describe('PullRequestPageTabs', () => {
   function setup(
-    { pullRequestPageFlagMultiSelect = false } = {
+    {
+      pullRequestPageFlagMultiSelect = false,
+      multipleTiers = false,
+      plan = TierNames.BASIC,
+      privateRepo = false,
+    } = {
       pullRequestPageFlagMultiSelect: false,
+      multipleTiers: false,
+      plan: TierNames.BASIC,
+      privateRepo: false,
     }
   ) {
     useFlags.mockReturnValue({
       pullRequestPageFlagMultiSelect,
+      multipleTiers,
     })
     server.use(
-      graphql.query('PullPageData', (req, res, ctx) =>
-        res(ctx.status(200), ctx.data(mockPullData))
-      ),
+      graphql.query('PullPageData', (req, res, ctx) => {
+        if (privateRepo) {
+          return res(ctx.status(200), ctx.data(mockPullDataPrivate))
+        }
+        return res(ctx.status(200), ctx.data(mockPullData))
+      }),
       graphql.query('GetCommits', (req, res, ctx) =>
         res(ctx.status(200), ctx.data(mockCommits))
       ),
       graphql.query('BackfillFlagMemberships', (req, res, ctx) =>
         res(ctx.status(200), ctx.data({}))
+      ),
+      graphql.query('OwnerTier', (req, res, ctx) =>
+        res(
+          ctx.status(200),
+          ctx.data({ owner: { plan: { tierName: plan.toLowerCase() } } })
```

Comment:
Nitpick: could we rename `plan` to `tier` ?

Reply:
I could at the query level, but it is called plan in the actual resolver so maybe we should also update it there and deprecate the plan resolver?

Reply:
Ah I should have been bit more specific, meant `plan: { tierName: tier.toLowerCase()}}`, cause tierName is a tier, not a plan itself

Reply:
Oh I see! Just to make sure I understand I need to replace the tiers with plans, so it needs to be

`plan: { planName: plan.toLowerCase()}}` ?